### ISSUE TITLE
error in loading BioData Catalyst platform projects in Desktop Rabix Composer for Mac

### ISSUE THREAD

**Issue by smgogarten** [rabix/composer#478]
When I launch Rabix Composer 1.5.0-rc1 using the "Edit in Rabix Composer" button on BioData Catalyst, I get a popup with this error:
```
Cannot read property 'id' of null

TypeError: Cannot read property 'id' of null
    at file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:3210667
    at Array.reduce (native)
    at t.project (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:3210622)
    at t._next (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:3189845)
    at t.OVmG.t.next (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:1467173)
    at t.notifyNext (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:3171784)
    at t.PIsA.t._next (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:1483745)
    at t.OVmG.t.next (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:1467173)
    at t._subscribe (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:3170458)
    at t.YaPU.e._trySubscribe (file:///Applications/rabix-composer.app/Contents/Resources/app.asar/main.d81da54260f60ee4ee75.bundle.js:1:1898498)
```

If I close the popup I can continue to edit my workflow, but clicking on "BioData Catalyst" under "My Projects" in the menu on the left generates the same error.

**Comment by kaushik-work** [comment]
Hi @smgogarten, does the next release candidate fix this issue?

**Comment by smgogarten** [comment]
No, I just installed rc.2 and got the same error. I should have done that first, but "Check for updates" in rc.1 told me everything was up to date.

**Comment by kaushik-work** [comment]
Hi @smgogarten sorry you are facing this issue and thank you for reporting it.

I was able to replicate this issue on our end, though it does not happen each time I do the operation.

**Comment by kaushik-work** [comment]
Closed by ddaa14fa94db5bf7562ebb1612916ec02c2fe200
